gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigResourceLocalHandler.java

```diff
@@ -188,6 +192,11 @@ public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig) thr
     return updateFlowConfig(flowId, flowConfig, true);
   }
 
+  @Override
+  public UpdateResponse partialUpdateFlowConfig(FlowId flowId, PatchRequest<FlowConfig> flowConfigPatch) throws FlowConfigLoggedException {
+    throw new UnsupportedOperationException("Partial update only supported by GobblinServiceFlowConfigResourceHandler");
```

Comment:
shouldn't partial update be supported in localhandler and not in gobblinhandler?

Reply:
As discussed, GaaS will call it in gobblin service handler and we want to reuse the logic for update in the gobblin service handler class. Since partial update should never be getting called from local handler we are just throwing exception to be clear.